shared/teams/delete-team/index.tsx

```diff
@@ -12,7 +12,7 @@ export type Props = {
 const Header = (props: Props) => (
   <>
     <Kb.Avatar teamname={props.teamname} size={64} />
-    <Kb.Icon type="icon-team-delete-28" style={{marginRight: -60, marginTop: -20}} />
+    <Kb.Icon type="icon-team-delete-28" style={{marginRight: -60, marginTop: -20, zIndex: 1}} />
```

Comment:
I noticed this in storybook, so just did the simplest thing following `really-leave` modal here https://github.com/keybase/client/blob/17cc9dda1052079051a51083d7ed2317009b6edb/shared/teams/really-leave-team/index.tsx#L35